spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -324,6 +338,11 @@ def main(argv: Sequence[str]):
         # add user parameters
         for paramStep in arguments.param:
             paramregmulti.addStep(paramStep)
+    elif label_type == 'rootlet':
+        # Add rootlet steps before step 1 and 2
+        step1.step = '2'
+        step2.step = '3'
+        paramregmulti = ParamregMultiStep([step0, step_rootlet, step1, step2])
```

Comment:
At first I was a little wary of manually inserting this step on behalf of the user, but I think it makes sense as long as we include the summary of what happens when `-lrootlets` is specified.